content/documentation/networks.mdx

```diff
@@ -5,7 +5,13 @@ description: Use Testnet | Iron Fish Documentation
 
 Iron Fish has 3 networks built into the SDK by default. These networks are useful for development and testing rather than trying to wait for someone to send you Mainnet $IRON.
 
-> **Note:** Once you start a node on a certain network, that node cannot be used on another network. Use `ironfish reset --networkId ...` to switch a node to another network.
+> **Note:** Once you start a node on a certain network, that node cannot be used on another network. We recommend using a different data directory for each network. Use `ironfish reset --networkId ...` to switch a node to another network.
```

Comment:
Is the data dir a concept we discuss much in the docs? If this is essentially the only mention of it, it feels like it might be more confusing than helpful. I wonder if we should link to the data dir page from here, at least

Reply:
you are right, not the best place to introduce this change also. I'll revert this